doc/tutorials/isingmodel.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<p>"using the"</p><p><br></p><p>maybe mention before that the base one is the NumPy one?</p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>